blogapp/spec/features/post_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  user = User.new(id: 1, name: 'Tom', photo: 'Link', bio: 'bio', posts_counter: 4)
+  user2 = User.new(id: 2, name: 'Lilly', photo: 'Link', bio: 'bio', posts_counter: 0)
+  post = Post.new(id: 1, author: user, title: 'Cars', text: 'text of first post', comments_counter: 1, likes_counter: 20)
+  post2 = Post.new(id: 2, author: user, title: 'Cars', text: 'text', comments_counter: 0, likes_counter: 20)
+  post3 = Post.new(id: 3, author: user, title: 'Cars', text: 'text', comments_counter: 0, likes_counter: 20)
+  post4 = Post.new(id: 4, author: user, title: 'Fourth post', text: 'text', comments_counter: 0, likes_counter: 20)
+  comment = Comment.new(author: user2, post:, text: 'text')
+  comment1 = Comment.new(author: user, post:, text: 'text')
+
+  before(:all) do
+    user.save
+    user2.save
+    post.save
+    post2.save
+    post3.save
+    post4.save
+    comment.save
+    comment1.save
+  end
+
+  after(:all) do
+    Post.destroy_all
+    User.destroy_all
+    Comment.destroy_all
+  end
+
+  describe 'Post view', type: :feature do
```

Comment:
- [OPTIONAL] Good work implementing tests as it's was required. I would suggest you implement some dynamically on them. For looking for the `@user.name`, instead of looking for a specific name like `Tom`, because sometimes the name can be different like in a real application